daprdocs/content/en/developing-applications/integrations/Azure/azure-kubernetes-service-extension.md

```diff
@@ -0,0 +1,89 @@
+---
+type: docs
+title: "Dapr extension for Azure Kubernetes Service (AKS)"
+linkTitle: "Azure Kubernetes Service (AKS) extension"
+description: "Provision Dapr on your Azure Kubernetes Service (AKS) cluster with the Dapr extension"
+weight: 4000
+---
+
+# Prerequisites
+- [Azure subscription](https://azure.microsoft.com/free/?WT.mc_id=A261C142F)
+- [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli-windows?tabs=azure-cli) and the ***aks-preview*** extension.
+- [Azure Kubernetes Service (AKS) cluster](https://docs.microsoft.com/azure/aks/tutorial-kubernetes-deploy-cluster?tabs=azure-cli)
+
+## Install Dapr using the AKS Dapr extension
+You have the option of using the AKS Dapr extension to provision Dapr on your AKS cluster. Additionally, the extension offers support for all native Dapr configuration capabilities through command-line arguments via the Azure CLI and offers the option of opting into automatic minor version upgrades of the Dapr runtime.
+
+{{% alert title="Note" color="warning" %}}
+If you install Dapr through the AKS extension, our recommendation is to continue using the extension for future management of Dapr instead of the Dapr CLI. Combining the two tools can cause conflicts and result in undesired behavior.
+{{% /alert %}}
+
+### How the extension works
+The Dapr extension uses the Azure CLI to provision the Dapr control plane on your AKS cluster. The Dapr control plane consists of:
+
+- **dapr-operator**: Manages component updates and Kubernetes services endpoints for Dapr (state stores, pub/subs, etc.)
+- **dapr-sidecar-injector**: Injects Dapr into annotated deployment pods and adds the environment variables DAPR_HTTP_PORT and DAPR_GRPC_PORT to enable user-defined applications to easily communicate with Dapr without hard-coding Dapr port values.
```

Comment:
"Injects Dapr into annotated deployment pods and adds the environment variables DAPR_HTTP_PORT and DAPR_GRPC_PORT. This enables user-defined applications to communicate with Dapr without the need to hard-code Dapr port values."